src/request_handler.js

```diff
@@ -98,6 +99,14 @@ export function oauthRequest (options, attempts = 1) {
     });
 }
 
+export function _awaitExponentialBackoff (attempts) {
```

Comment:
It would be nice to have a test for this, but doing a full integration test would require us to have a reliable way of getting 503 errors ~~(and it seems impolite to use `/search` for that purpose)~~.

Can you add something that just tests `r._awaitExponentialBackoff`?